manifests/cf-manifest/operations.d/300-upgrade-cflinuxfs2-for-cve-mitigation.yml

```diff
@@ -0,0 +1,9 @@
+- path: /releases/name=cflinuxfs2/url
+  type: replace
+  value: https://storage.googleapis.com/cf-deployment-compiled-releases/cflinuxfs2-1.216.0-ubuntu-trusty-3586.16-20180612-151850-131406428.tgz
```

Comment:
Why are we using the compiled release for this? We should be sticking to the standard releases like we do elsewhere.

Reply:
Oh, I thought we used the compiled releases now to speed things up?

Reply:
Ah, [only for BOSH](https://www.pivotaltracker.com/story/show/155514398).

Reply:
Fixed, but will need to re-test the deployment.